api-schema/src/main/java/org/stellar/anchor/api/asset/AssetInfo.java

```diff
@@ -0,0 +1,27 @@
+package org.stellar.anchor.api.asset;
+
+import org.stellar.anchor.api.sep.AssetInfo.Schema;
+import org.stellar.anchor.api.sep.sep31.Sep31InfoResponse;
+import org.stellar.anchor.api.sep.sep38.InfoResponse;
+
+public interface AssetInfo {
+
+  Schema getSchema();
+
+  /**
+   * Returns the asset identification name following the structure scheme of scheme:identifier
+   * The currently accepted scheme values are:
+   * - stellar: Used for Stellar assets. The identifier follows the SEP-11 asset format <Code:IssuerAccountID>.
+   * - iso4217: Used for fiat currencies. The identifier follows the ISO 4217 three-character currency code.
+   * For example:
+   * - Stellar USDC would be identified as: stellar:USDC:GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN
+   * - Fiat USD would be identified as: iso4217:USD
+   *
+   * @return A string representing the asset identification name, formatted as <scheme>:<identifier>.
+   */
+  String getAssetIdentificationName();
+
+  InfoResponse.Asset toSEP38InfoResponseAsset();
```

Comment:
What's the advantage of implementing this in the interface over adding the conversion function in the [sep38.InfoResponse](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/api-schema/src/main/java/org/stellar/anchor/api/sep/sep38/InfoResponse.java)?

Reply:
I think this function should be part of the asset itself (or maybe helper function) instead of going to service, it's much clear to maintain if we want to add new asset schema, or add new fields to existing schema, in the future

Reply:
We will still need to implement the conversion method either, whether that is in the `AssetInfo` class or `InfoResponse`. I'm not sure I understand the advantage of implementing the method here. Since we will be using this method only in the SEP-38 code, I would prefer if we move it there as well.

Reply:
I would feel more comforable to move the method to the service classes because conversion is part of the business logic.